apps/docs/src/guide/transactions/index.md

```diff
@@ -0,0 +1,13 @@
+# Transactions
+
+A transaction is any interaction with the blockchain. This is the primary way of interacting with the Fuel Network and can include actions like transferring assets, deploying contracts and minting tokens.
+
+Within the Fuel Network, we have the following transaction types:
+
+- Script
+- Create
+- Mint
+
+The SDK provides class helpers for handling script and create transactions, `ScriptTransactionRequest` and `CreateTransactionRequest` respectively. Mint transactions are recognized by the SDK but can only be created by the block producer so do not have use any use outside of block creation. Therefore there is no need to manually create them.
```

Comment:
```suggestion
The SDK provides class helpers for handling script and create transactions: `ScriptTransactionRequest` and `CreateTransactionRequest`, respectively. 
Mint transactions can only be created by the block producer and do not have any use outside of block creation. Therefore, the SDK only provides the ability to decode them.
```

If this rewording doesn't go through, there is an additional "use" in the original wording that should be removed.